src/osip-monitoring-controller/src/main/resources/Bundle.properties

```diff
@@ -19,3 +19,23 @@ monitoring.tank.progress.fillLevel=Fill level
 monitoring.tank.progress.temperature=Temperature
 monitoring.mixTank.motorSpeed=Motor speed
 monitoring.mixTank.color=Color
+monitoring.helpdialog.title=Help
+monitoring.helpdialog.mainTab.header=Shell
+monitoring.helpdialog.mainTab.introduction=The OSIP monitoring shell can be used to monitor the tanks, pipes and alarms of the production site simulation via the OPC UA protocol.
+monitoring.helpdialog.mainTab.layoutSection=Shell Layout
+monitoring.helpdialog.mainTab.layoutText=The layout of the monitoring shell can be divided into five parts. At first there are the information about the upper tanks of the production site. These boxes are located at the upper side of the shell and are bordered in the color of the liquid in the tank. At the right of those parts, there is the information about the mix tank. The color is shown within the box. Underneath those boxes there is a textfield which is used to print alarm messages.
+monitoring.helpdialog.mainTab.alarmSection=Alarms
+monitoring.helpdialog.mainTab.alarmText=Every tank box has four possible alarms which are shown as small red, green or grey circles at the upper left corner of the box. If the circle is green the alarm won't be active, if it is red then the alarm will be active. In case, the circle is gray, the alarm is disabled. Additionally there is a big red and green circle at the bottom right of the shell. If the red circle is filled, some alarm will be active and if the green circle is filled, no alarm will be active. When both circles are gray, every alarm will be disabled.
```

Comment:
Every tank box has four possible alarms which are shown as small red, green or grey **[Ich habe recherchiert und *gray* kommt am häufigsten im AE vor. Hier ist die einzige Stelle, an der *grey* vorkommt.]** circles at the upper left corner of the box. If the circle is green **[Komma]** the alarm won't be active, if it is red **[Komma]** then the alarm will be active. In case, the circle is gray, the alarm is disabled. Additionally **[Komma]** there is a big red and green circle at the bottom right of the shell. If the red circle is filled, some alarm will be active and if the green circle is filled, no alarm will be active. When both circles are gray, every alarm will be disabled.